### ISSUE TITLE
gnocchi-api does not work with packaged uwsgi

### ISSUE THREAD

**Issue by jd** [gnocchixyz/gnocchi#478]
gnocchi-api does not work with uwsgi provided by Ubuntu and probably other distros. On this OS it is built dynamically with modules that are not loaded by default (e.g. http, python…).

We should be able to detect that and load uwsgi with the right options to make it work.

**Comment by Asu4ni** [comment]
Yes, I encountered the same problems on ubuntu 16.04 as well. So I used `pip` to install uwsgi at the end.
